extensions/wrapper/wrapper-api/src/main/java/de/sovity/edc/ext/wrapper/api/usecase/model/CatalogQueryParams.java

```diff
@@ -0,0 +1,24 @@
+package de.sovity.edc.ext.wrapper.api.usecase.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@RequiredArgsConstructor
+@Schema(description = "Catalog query parameters")
+public class CatalogQueryParams {
+    @Schema(description = "Target EDC DSP endpoint URL", requiredMode = Schema.RequiredMode.REQUIRED)
+    private String targetEdc;
```

Comment:
needs to have "connectorEndpoint" in it, e.g. targetEdcConnectorEndpoint or connectorEndpoint